### PR TITLE
Add quiz card CSS and create HTML

### PR DESCRIPTION
**Pull request by frederikebruendermann**

### CODE REVIEW

File: src/styles/quiz-card/quiz-card.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<section class="card">
+  <h2 class="card__heading">Frage</h2>
+  <p class="card__question">Wie viele Zähne hat ein Eisbär?</p>
+  <button class="card__button">Zeige mir die Antwort</button>
+  <ul class="card__tag-list">
+    <li card__tag-tag>32</li>
+    <li card__tag-tag>42</li>
+    <li card__tag-tag>52</li>
+    <li card__tag-tag>62</li>
```
Comment (macfrei) [inline src/styles/quiz-card/quiz-card.html]: Did you mean: also, I'd probably name the class `card__tag`.
```suggestion
    <li class="card__tag">32</li>
    <li class="card__tag">42</li>
    <li class="card__tag">52</li>
    <li class="card__tag">62</li>
```

Comment (macfrei) [reply src/styles/quiz-card/quiz-card.html]: It also looks a bit as if the tags are your answer. The tags should be an additional feature, kind of like hashtags. The answer would also be a `p`.

File: src/styles/quiz-card/quiz-card.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+.card__tag-list {
+  list-style: square;
+  display: flex;
+  flex-flow: column wrap;
+  line-height: 2rem;
+  margin: 15px 0;
+}
+
+.card__tag-tag {
```
Comment (macfrei) [inline src/styles/quiz-card/quiz-card.css]:
```suggestion
.card__tag {
```

File: src/styles/_utility.css [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
-
 .m0 {
   margin: 0 !important;
 }
+
+.df {
+  display: flex;
+  justify-content: space-between;
+  list-style-type: none;
+}
```
Comment (macfrei) [inline src/styles/_utility.css]: I think this does a bit too much for a utility class. I guess `df` stands for `display: flex`? If so, I would leave out the `list-style-type` as it has nothing to do with a flexbox.

File: src/index.html [hunk 4]
```diff
@@ -1,15 +1,31 @@
...
+      <h2 class="card__question">Frage</h2>
+      <button class="card__button">Zeige mir die Antwort</button>
+      <p class="card__answer">Wie viele Zähne hat ein Eisbär?</p>
+      <ul class="card__tag-list">
+        <li>32</li>
+        <li>42</li>
+        <li>52</li>
+        <li>62</li>
+      </ul>
+    </section>
```
Comment (macfrei) [inline src/index.html]: The main content of your website should be wrapped in a main element.
```suggestion
   <main>
    <section class="card">
      <h2 class="card__question">Frage</h2>
      <button class="card__button">Zeige mir die Antwort</button>
      <p class="card__answer">Wie viele Zähne hat ein Eisbär?</p>
      <ul class="card__tag-list">
        <li>32</li>
        <li>42</li>
        <li>52</li>
        <li>62</li>
      </ul>
    </section>
  </main>
```

**Review by macfrei — COMMENTED** [review]
Well done! Added a few comments 👍

File: src/styles/quiz-card/quiz-card.html (hunk 1 above)
Comment (frederikebruendermann) [reply src/styles/quiz-card/quiz-card.html]: Oh yes, you are right! Thank you

**Review by frederikebruendermann — COMMENTED** [review]
Update done regarding the requested changes.  Thx!

**Pull request merged**